packages/@okta/vuepress-theme-prose/global-components/ApiLifecycle.vue

```diff
@@ -30,12 +33,14 @@ export default {
   props: {
     access: {
       type: String,
-      default: "beta"
+      default: "Beta"
     }
   },
   computed: {
     link() {
-      return this.access === this.labelType.IDENTITY_ENGINE ? IE_LINK : DEFAULT_LINK;
+      return this.access === this.labelType.IDENTITY_ENGINE ||
+             this.access === this.labelType.CLASSIC_ENGINE
```

Comment:
Is this logic right? If we show the Classic Engine label, we use the Identity Engine link URL?

Reply:
Perhaps we should consider leaving a comment here explaining and also linking to the ticket